.github/workflows/test_odin_control.yml

```diff
@@ -33,10 +33,10 @@ jobs:
         fi
         coverage xml --rcfile=$COVERAGE_RC
     - name: Upload coverage to Codecov
-      uses: codecov/codecov-action@v1
+      uses: codecov/codecov-action@v3
       with:
         name: ${{ matrix.python-version }}
-        fail_ci_if_error: true
```

Comment:
Was this change made purposefully?

Reply:
Yes, to stop failures in `codecov.io` coverage data uploads from flagging the CI actions as test failures :wink: They will still fail any tests fail.